src/App.js

```diff
@@ -81,6 +82,7 @@ class App extends Component {
   state = { appState: AppState.currentState };
 
   async componentDidMount() {
+    initApolloCache();
```

Comment:
I think there's no guarantee that this will run before the queries if you initialize it here.